tools/RestfulApiWebService/OperationOnDB/Filters/BasicAuthenticationAttribute.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
```

Comment:
one is mvc, another is for web api, there are a little differences

---
In reply to: [131039250](https://github.com/dotnet/docfx/pull/1906#discussion_r131039250) [](ancestors = 131039250)